README.md

```diff
@@ -16,7 +16,7 @@
 - [License](#license)
 
 ## Introduction
-The purpose of the PathFinder smart vehicle is to offer to public transportation companies and their client a digitalized experience. We aim to do so by offering:
+The purpose of the PathFinder smart vehicle is to offer to public transportation companies, and their client a digitalized experience. We aim to do so by offering:
```

Comment:
Minor change suggestion:

The purpose of the PathFinder smart vehicle is to offer a digitalized experience to public transportation companies and their customers.

Reply:
Done